examples/Producer/Producer.csproj

```diff
@@ -9,8 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!-- nuget package reference: <PackageReference Include="Confluent.Kafka" Version="1.9.0-RC7" /> -->
```

Comment:
this is an actual change, not sure what it means

Reply:
oh whoops, thanks.